_rules/button-accessible-name-97a4e1.md

```diff
@@ -117,41 +117,41 @@ Off screen elements should be tested.
 
 #### Passed Example 8
 
-Button has [accessible name][] that is not only [whitespace][].
+Some buttons have a default value. In this example the `input` element has "reset" as the [accessible name][], or translated of "reset" provided by the browser.
 
 ```html
-<button>:-)</button>
+<input type="reset" />
 ```
 
-#### Passed Example 9
+### Failed
 
-Input button has an [accessible name][] that comes from the default "reset" text.
+#### Failed Example 1
+
+The `button` element has an empty [accessible name][], because it has no content, nor any attribute that would give it an accessible name.
```

Comment:
```suggestion
The `button` element has an empty [accessible name][], because it has no content, nor any attribute that would give it an [accessible name][].
```